metadata-ingestion/src/datahub/configuration/kafka.py

```diff
@@ -16,6 +16,10 @@ class _KafkaConnectionConfig(ConfigModel):
         description="Extra schema registry config serialized as JSON. These options will be passed into Kafka's SchemaRegistryClient. https://docs.confluent.io/platform/current/clients/confluent-kafka-python/html/index.html?#schemaregistryclient",
     )
 
+    list_topics_timeout_seconds: int = Field(
```

Comment:
can we rename this to something generalized? 

`client_timeout_seconds`. We will use this for another upcoming use case in the kafka source!